src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea {
+    public abstract double areaCalculator();
```

Comment:
The method inside the interface should not have 'public abstract' modifiers as they are implied by default in an interface context. You can simply define the method as 'double areaCalculator();'.